lldb/source/Host/posix/MainLoopPosix.cpp

```diff
@@ -155,7 +181,9 @@ Status MainLoopPosix::RunImpl::Poll() {
     void *sigset_ptr;
     size_t sigset_len;
   } extra_data = {&kernel_sigset, sizeof(kernel_sigset)};
-  if (syscall(__NR_pselect6, nfds, &read_fd_set, nullptr, nullptr, nullptr,
+  if (syscall(__NR_pselect6, nfds, &read_fd_set, /*writefds=*/nullptr,
```

Comment:
pselect is sufficiently complex that you might want to add a small blurb what you're doing

Reply:
I think I can do something even better ~~> #115197

(I'll rebase this PR on top of that afterwards)